test/index.test.js

```diff
@@ -141,6 +142,40 @@ describe('Bandit-linter', () => {
       }))
     })
 
+    // This event happens when there is already opened pull request which
+    // happens across forks. Then when there is a new commit to this pull request
```

Comment:
```suggestion
    // happens across forks. Then a new commit to this pull request,
```

Reply:
I have other suggestion. I will commit it.